Source/Snowman.cpp

```diff
@@ -188,13 +203,15 @@ void Snowman::translateSnowman(GLFWwindow* window, const bool& shift, bool& canM
 
 	// Snowman Translation by WASD keys
 	float snowmanTranslationSpeed = globalSpeed * 0.03f;
-	bool moveLeft = glfwGetKey(window, GLFW_KEY_A) == GLFW_PRESS;
+	bool moveLeft  = glfwGetKey(window, GLFW_KEY_A) == GLFW_PRESS;
 	bool moveRight = glfwGetKey(window, GLFW_KEY_D) == GLFW_PRESS;
-	bool moveUp = glfwGetKey(window, GLFW_KEY_W) == GLFW_PRESS;
-	bool moveDown = glfwGetKey(window, GLFW_KEY_S) == GLFW_PRESS;
+	bool moveUp    = glfwGetKey(window, GLFW_KEY_W) == GLFW_PRESS;
+	bool moveDown  = glfwGetKey(window, GLFW_KEY_S) == GLFW_PRESS;
+
+	adjustKeysToRotation();
 
 	// Move snowman left (+ x direction)
-	if (moveLeft) {
+	if (moveLeft && collideLeftToRight) {
```

Comment:
the naming of this variable doesn't make sense in this context to me. If I wish to moveLeft, why would a left to right collision have to do with allowing me to move left?